IFC4x3/Properties/r/RatedFrequency_0xk7ZvULjDVACUV9Qs5Q4T/Documentation.md

```diff
@@ -0,0 +1 @@
+Frequency of the AC electric power system that the Ripple Control supplied reaches its optimum operating condition
```

Comment:
Something is bothering me with this description, It seems of. Why is "Ripple Control" capitalized?
```suggestion
Frequency of the AC electric power system that the Ripple Control supplied reaches its optimum operating condition.
```